content/docs/integrations/github-discovery/index.md

```diff
@@ -10,7 +10,7 @@ integrationType: Integration
 
 ## Introduction
 
-Roadie can load your catalog files automatically from GitHub.
+Roadie can load your catalog files automatically from GitHub. It takes advantage of the github webhooks to keep the catalog up to date with your additions and modifications.
 
```

Comment:
If this is going to be enabled by default, then the goal of the document should be to teach the user how to change or add to the default settings.

I think we can do more in the intro to help the user understand what they will get out of reading this document. Why would I want to read it? What will I learn?